### PR TITLE
Test header variable for subdomains

### PR DESCRIPTION
**Pull request by jhonny218**
## Description
Adding a console.log to check the Facility locator link inside the header for subdomains

## Testing done
Will be tested live since no local testing for subdomain is available

## Screenshots
Screen Shot 2020-05-08 at 11 14 16 AM

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/platform/site-wide/user-nav/components/HelpMenu.jsx [hunk 1]
```diff
@@ -16,6 +16,13 @@ function HelpMenu({ clickHandler, cssClass, isOpen }) {
     : FACILITY_LOCATOR_URL;
   const icon = <IconHelp color="#fff" role="presentation" />;
 
+  // eslint-disable-next-line no-console
+  console.log(
+    isVATeamSiteSubdomain(),
+    facilityLocatorUrl,
+    FACILITY_LOCATOR_URL,
+  );
```
Comment (ncksllvn) [inline src/platform/site-wide/user-nav/components/HelpMenu.jsx:24]: I think the issue (if I understand correctly) is actually on line 8. We have a hardcoded list of VA subdomains here - `src/platform/utilities/environment/va-subdomain.js`. I think that function should be changed to return true if the domain isn't `(www|staging|dev).va.gov`.  think it's returning false for subdomains not listed, leading to the Facility Locator appearing relative to the subdomain.

Comment (jhonnyoddball) [reply src/platform/site-wide/user-nav/components/HelpMenu.jsx:24]: I added all the new subdomains already but it didn't work in this [PR](https://github.com/department-of-veterans-affairs/vets-website/pull/12558)

Comment (ncksllvn) [reply src/platform/site-wide/user-nav/components/HelpMenu.jsx:24]: Ah okay. I think it does work actually, but didn't make it into the release?

Comment (jhonnyoddball) [reply src/platform/site-wide/user-nav/components/HelpMenu.jsx:24]: It did, so I'm not sure. I'm going to add your suggestion and maybe add an unit test for it

Comment (ncksllvn) [reply src/platform/site-wide/user-nav/components/HelpMenu.jsx:24]: Oh, for some reason the commits aren't listed under yesterday's release. Weird

Comment (ncksllvn) [reply src/platform/site-wide/user-nav/components/HelpMenu.jsx:24]: I copy/pasted `isVATeamSiteSubdomain` into the console on that domain and it returned true, so I don't think that's the problem

Comment (ncksllvn) [reply src/platform/site-wide/user-nav/components/HelpMenu.jsx:24]: I actually don't think it was deployed yesterday. It looks like it failed - https://dsva.slack.com/archives/C0MQ281DJ/p1588875279132900. A content deployment happened later but that wouldn't have published this.

Comment (jhonnyoddball) [reply src/platform/site-wide/user-nav/components/HelpMenu.jsx:24]: I did it too and it worked...

Comment (ncksllvn) [reply src/platform/site-wide/user-nav/components/HelpMenu.jsx:24]: I don't think yesterday's release went out. I think the header on patientsafety.va.gov will work after today's deployment.

**Pull request closed without merging**